smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -0,0 +1,192 @@
+  File aligned_bam
+  File gtf
+  String fc_out
+  
+  command {
+    featureCounts -s 0 -t exon -g gene_id -p -B -C -a ${gtf} -o "${fc_out}.gene.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g transcript_id -p -B -C -a ${gtf} -o "${fc_out}.transcripts.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g exon_id -p -B -C -a ${gtf} -o "${fc_out}.exon.counts.txt" ${aligned_bam}
+  }
+  runtime {
+    docker:"humancellatlas/star_dev:v1"
+    memory: "10 GB"
+    disks :"local-disk 50 HDD"
+  }
+  output {
+    File genes="${fc_out}.gene.counts.txt"
+    File exons="${fc_out}.exon.counts.txt"
+    File trans="${fc_out}.transcripts.counts.txt"
+  }
+}
+
+task RsemExpression {
+  File trans_aligned_bam
+  File rsem_genome
+  String rsem_out
+  
+  command {
+    tar -zxvf ${rsem_genome}
+    echo "Aligning fastqs and calculating expression"
+    rsem-calculate-expression --bam --paired-end ${trans_aligned_bam} rsem/rsem_trans_index  "${rsem_out}"
+    ## parise gene expected_count out
+    cut -f 1,4,5 "${rsem_out}.genes.results" >"${rsem_out}.gene.expected_counts"
+  }
+  runtime {
+    docker: "humancellatlas/rsem"
+    memory: "10 GB"
+    disks: "local-disk 100 HDD"
+  }
+  output {
+    File rsem_gene="${rsem_out}.genes.results"
+    File rsem_transc="${rsem_out}.isoforms.results"
+    File rsem_gene_count="${rsem_out}.gene.expected_counts"
+   }
+}
+
+task CollectAlignmentSummaryMetrics {
+	File aligned_bam
+  File ref_genome_fasta
+  String output_filename
+  
+  command {
+		java -Xmx10g -jar /usr/picard/picard.jar CollectAlignmentSummaryMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      METRIC_ACCUMULATION_LEVEL=ALL_READS \
+      INPUT=${aligned_bam} \
+      OUTPUT="${output_filename}" \
+      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      ASSUME_SORTED=true
+  }
+  output {
+    File alignment_metrics ="${output_filename}"
+  }
+  runtime {
+    docker:"humancellatlas/picard"
+    memory:"10 GB"
+    disks: "local-disk 10 HDD"
+  }
+}
+
+task CollectRnaSeqMetrics {
+	File aligned_bam
+  File ref_genome_fasta
+  File rrna_interval
+  String output_filename
+  File ref_flat
+  
+  command {
+    java -Xmx10g -jar /usr/picard/picard.jar  CollectRnaSeqMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      REF_FLAT=${ref_flat} \
+      RIBOSOMAL_INTERVALS=${rrna_interval} \
+      INPUT=${aligned_bam} \
+      OUTPUT="${output_filename}" \
+      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      ASSUME_SORTED=true \
+      STRAND_SPECIFICITY=NONE
+  }
+  output {
+     File rna_metrics="${output_filename}"
+  }
+  runtime {
+		docker:"humancellatlas/picard"
+		memory:"10 GB"
+		disks: "local-disk 10 HDD"
+  }
+}
+
+workflow Ss2RsemSingleSample {
+  File fastq_read1
+  File fastq_read2
+  File gtf
+  File ref_fasta
+  File rrna_interval
+  File ref_flat
+  String star_genome
+  String output_prefix
+  String rsem_genome
+  
+  call Star {
+    input:
+    input_fastq_read1 = fastq_read1,
```

Comment:
Can the list of inputs be indented once more in all of these calls? i.e.
```
input:
      input_bam = GatherBamFiles.output_bam,
      input_bam_index = GatherBamFiles.output_bam_index,
      output_bam_prefix = base_file_name
}```

Reply:
Yes please, two spaces indent under "input:"